src/main/java/org/folio/circulation/domain/validation/ItemStatusValidator.java

```diff
@@ -40,14 +43,16 @@ public ItemStatusValidator(
     Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
 
     return loanAndRelatedRecords
-      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.MISSING))
-      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.DECLARED_LOST));
+      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, MISSING))
+      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, DECLARED_LOST));
   }
 
-  public Result<LoanAndRelatedRecords> refuseItemWithDeniedStatuses(
+  public Result<LoanAndRelatedRecords> refuseWhenItemIsNotAllowedForCheckOut(
     Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
 
-    return refuseWhenItemIsMissing(loanAndRelatedRecords)
-      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.CLAIMED_RETURNED));
+    return loanAndRelatedRecords
+      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, MISSING))
```

Comment:
I know this predates this change, Could `checkAndRefuseItem` read better as `refuseWhenItemIs`

```suggestion
      .next(p -> refuseWhenItemIs(loanAndRelatedRecords, MISSING))
```